meta-lmp-base/recipes-kernel/wireguard/wireguard-module_%.bbappend

```diff
@@ -5,6 +5,6 @@ python __anonymous() {
     pprovider = d.getVar("PREFERRED_PROVIDER_virtual/kernel")
     if pprovider != "linux-lmp" and pprovider != "linux-lmp-lts" and pprovider != "linux-lmp-rt" \
             and pprovider != "linux-lmp-dev" and pprovider != "linux-lmp-fslc-imx" \
-            and d.getVar("KERNEL_BUILTIN_WIREGUARD") != "1":
+            and d.getVar("KERNEL_BUILTIN_WIREGUARD") == "0":
```

Comment:
We can probably remove the lines above as well, and just look for KERNEL_BUILTIN_WIREGUARD.

Reply:
right, whenever the variable KERNEL_BUILTIN_WIREGUARD is not defined as "0" the builtin kernel module will be used